cmd/kubelet/app/server.go

```diff
@@ -655,6 +656,16 @@ func RunKubelet(kcfg *KubeletConfig) error {
 
 	util.ApplyRLimitForSelf(kcfg.MaxOpenFiles)
 
+	// TODO(dawnchen): remove this once we deprecated old debian containervm images.
+	// This is a workaround for issue: https://github.com/opencontainers/runc/issues/726
+	// The current chosen number is consistent with most of other os dist.
+	var path = "/proc/sys/kernel/keys/root_maxkeys"
+	glog.Infof("Setting $path to %d", path, 1000000)
+	err = ioutil.WriteFile(path, []byte(fmt.Sprintf("%d", uint32(1000000))), 0644)
+	if err != nil {
+		return fmt.Errorf("failed to update %s: %v", path, err)
+	}
```

Comment:
Also we do the similar thing for other daemons, like kube_proxy: https://github.com/kubernetes/kubernetes/blob/dae5ac482861382e18b1e7b2943b1b7f333c6a2a/cmd/kube-proxy/app/conntrack.go#L42